mlrun/model_monitoring/features_drift_table.py

```diff
@@ -0,0 +1,570 @@
+        )
+        sub_header_table = go.Table(
+            header={
+                "values": sub_headers,
+                "align": "center",
+                "font": {"size": self._FONT_SIZE, "color": self._FONT_COLOR},
+                "line": {"color": self._SEPARATORS_COLOR},
+            },
+            columnwidth=(
+                [self._FEATURE_NAME_COLUMN_WIDTH]
+                + [self._VALUE_COLUMN_WIDTH]
+                * (2 * len(self._statistics_columns) + len(self._metrics_columns))
+                + [self._HISTOGRAMS_COLUMN_WIDTH, self._NOTIFICATIONS_COLUMN_WIDTH]
+            ),
+            header_fill_color=self._BACKGROUND_COLOR,
+        )
+
+        return header_table, sub_header_table
+
+    def _separate_feature_name(self, feature_name: str) -> List[str]:
+        """
+        Separate the given feature name by the maximum length configured in the class. Used for calculating the amount
+        of lines required to represent the longest feature name in the table, so the row heights will fit accordingly.
+
+        :param feature_name: The feature name to separate.
+
+        :return: The feature name's list of the separations.
+        """
+        return [
+            feature_name[i : i + self._FEATURE_NAME_MAX_LENGTH]
+            for i in range(0, len(feature_name), self._FEATURE_NAME_MAX_LENGTH)
+        ]
+
+    def _get_values_formats(self, values: List[Union[str, int, float]]) -> List[str]:
+        """
+        Plotly uses D3 formatter to format values. This method return the list of formats according to the configured
+        properties in the class to the given row of values - list of feature statistics and metrics values.
+
+        :param values: Feature row values.
+
+        :return: List of all the formats to use for the given row.
+        """
+        # Initialize the formats per value list:
+        formats = []  # type: List[str]
+
+        # Start going over the values:
+        for value in values:
+            if isinstance(value, int):
+                # An integer should be parsed into short characters (e.g: 10000 -> 10k, 1100000 -> 1.1m)
+                formats.append("~s")
+            elif isinstance(value, float):
+                if value.is_integer():
+                    # A float with an integer value should be parsed like an integer:
+                    formats.append("~s")
+                else:
+                    integer_length = len(str(int(value)))
+                    if integer_length > self._VALUES_MAX_DIGITS:
+                        # If the integer value of the float is that long, parse like an integer:
+                        formats.append("~s")
+                    else:
+                        # Round it to the remaining digits left considering the integer length:
+                        formats.append(f".{self._VALUES_MAX_DIGITS - integer_length}")
+            else:
+                # A string do not need reformatting:
+                formats.append("")
```

Comment:
I believe it would be cleaner to create a helper method like so and use it in the for loop:
```
def _get_value_parse_character(self, value):
    # A string do not need reformatting:
    if isinstance(value, str):
        return ""
    
    integer_length = len(str(int(value)))

    # Any whole number or number with a long integer value should be parsed into 
    # short characters (e.g: 10000 -> 10k, 1100000 -> 1.1m)
    if value % 1 == 0 or integer_length > self._VALUES_MAX_DIGITS:
        return "~s"
    
    # Round it to the remaining digits left considering the integer length:
    return f".{self._VALUES_MAX_DIGITS - integer_length}"
```

Reply:
done deal :+1: